examples/aws_ebs/README.md

```diff
@@ -0,0 +1,17 @@
+This is a simple web server pod which serves HTML from an AWS EBS
+volume.
+
+Create a volume in the same region as your node add your volume
+information in the pod description file aws-ebs-web.yaml then create
+the pod:
+
+  $ kubectl create -f aws-ebs-web.yaml
```

Comment:
Could you wrap the shell stuff in Github's fenced code blocks? E.G.

<pre><code>```shell
$ kubectl create -f aws-ebs-web.yaml
```</code></pre>